taichi/rhi/vulkan/vulkan_device.cpp

```diff
@@ -1329,6 +1329,7 @@ struct VulkanDevice::ThreadLocalStreams {
 VulkanDevice::VulkanDevice()
     : compute_streams_(std::make_unique<ThreadLocalStreams>()),
       graphics_streams_(std::make_unique<ThreadLocalStreams>()) {
+  caps_.set(DeviceCapability::spirv_version, 0x10000);
```

Comment:
Is this newly enforced btw?

Reply:
that's the minimum requirement. That capacity is required for the backend to work iirc